go-controller/pkg/clustermanager/egressip_controller.go

```diff
@@ -690,35 +690,42 @@ func (eIPC *egressIPClusterController) reconcileNonOVNNetworkEIPs(node *v1.Node)
 	if err != nil {
 		return fmt.Errorf("unable to list EgressIPs, err: %v", err)
 	}
+	reconcileEgressIPs := make([]*egressipv1.EgressIP, 0)
```

Comment:
NIT: since we will be holding the lock, maybe we should have the capacity of the slice set here, so there is no delay in having to grow it:
`reconcileEgressIPs := make([]*egressipv1.EgressIP, 0, len(egressIPs.Items))`